src/main/java/uk/ac/cam/cl/dtg/isaac/quiz/IsaacNumericValidator.java

```diff
@@ -208,12 +208,13 @@ public final QuestionValidationResponse validateQuestionResponse(
         } catch (NumberFormatException e) {
             log.debug("Validation failed for '" + answerFromUser.getValue() + " " + answerFromUser.getUnits() + "': "
                     + "cannot parse as number!");
-
-            String feedback = "Your answer is not in a format we recognise, please enter your answer as a decimal number.";
```

Comment:
I think we should still keep this feedback for this error and not use the default validation response as you've changed to below.
Given you override it in the frontend now, it doesn't matter so much, but it seems silly to add a less useful error message instead of this one.